lib/display_board.rb

```diff
@@ -1,2 +1,11 @@
 # Define display_board that accepts a board and prints
 # out the current state.
+def display_board(board = [" "," "," "," "," "," "," "," "," "])
+  puts " " + board[0]+ " | " + board [1] + " | " + board[2] + " "
```

Comment:
Try to use interpolation `#{}` instead of concatenation `+` for strings with dynamic variable values.

```ruby
puts " " + board[0]+ " | " + board [1] + " | " + board[2] + " "
```

vs 

```ruby
puts "  #{board[0]} | #{board[1]} | #{board[2]} "
```